src/components/incident/IncidentDetails.tsx

```diff
@@ -98,14 +98,10 @@ type TagChipPropsType = {
 };
 
 const isValidUrl = (url: string) => {
-  // Pavlo's answer at
-  // https://stackoverflow.com/questions/5717093/check-if-a-javascript-string-is-a-url
-  try {
-    new URL(url);
-  } catch (_) {
-    return false;
-  }
-  return true;
+  let pattern = new RegExp(
+    "^(http(s)?:\\/\\/)+[\\w\\-\\._~:\\/?#[\\]@!\\$&'\\(\\)\\*\\+,;=.]+$",
+    'i');
```

Comment:
Bare regexps are a code smell. If you got this one from somewhere else, citation needed. If not, you need a *large* test-suite documenting the urls this regexp was designed for: With path, with query, with php-style queries etc.